frontend/src/Containers/TodayContainer.tsx

```diff
@@ -4,9 +4,22 @@ import {
   NoDeliveries,
 } from "../Components/Today/DeliveryTimeline";
 import "../Styles/Delivery.css";
+import { getVolunteerTasks } from "../services";
+import { useState, useEffect } from "react";
 
 const TodayContainer = () => {
-  let noDeliveries = false;
+  let volunteerId = 5;
+
+  const [noDeliveries, setNoDeliveries] = useState(false);
+
+  //if there isn't a task assigned to this volunteer for today
+  useEffect(() => {
+    //todo get the volunteer task and check for today's date
```

Comment:
could you implement this todo? it should be a part of this PR since we want to fully connect to the backend